main_test.go

```diff
@@ -129,14 +129,14 @@ func TestMain(t *testing.T) {
 		_ = os.Remove(file.Name())
 	}()
 
-	event := types.FixtureEvent("entity1", "check1")
+	event := corev2.FixtureEvent("entity1", "check1")
 	eventJSON, _ := json.Marshal(event)
 	_, err := file.WriteString(string(eventJSON))
 	require.NoError(t, err)
 	require.NoError(t, file.Sync())
 	_, err = file.Seek(0, 0)
 	require.NoError(t, err)
-	stdin = file
+	os.Stdin = file
```

Comment:
It's safer for test stability to avoid reading/writing files unless absolutely necessary (see https://github.com/sensu/sensu-plugins-go-library/pull/5#discussion_r286107307). Would it be possible to stream these bytes to stdin instead?

Reply:
While you certainly make sense, but I am finding it difficult to stream bytes to os.Stdin, while is of type *File. Since, this is something which was pre-existing and not part of the change done in this PR, can it be taken care of by some other pull request?